scripts/invariant_checks/checks/idled-deployments-with-more-than-0-replicas.js

```diff
@@ -0,0 +1,25 @@
+function isIdled(deployment) {
+  return deployment.metadata.labels && deployment.metadata.labels["mojanalytics.xyz/idled"] == "true"
```

Comment:
If we're sure this is coming out as a string (and it makes sense as they're labels) I may use the `===` to be on the safe side considering how weird JS can be.